python_scripts/trees.py

```diff
@@ -81,19 +81,19 @@
 # We can first check the feature distributions by looking at the diagonal plots
 # of the pairplot. We can build the following intuitions:
 #
-# * The Adelie specie is separable from the Gentoo and Chinstrap species using
+# * The Adelie species is separable from the Gentoo and Chinstrap species using
 #   the culmen length;
-# * The Gentoo specie is separable from the Adelie and Chinstrap species using
+# * The Gentoo species is separable from the Adelie and Chinstrap species using
 #   the culmen depth.
 #
-# ## How decision tree are built?
+# ## How are decision tree built?
 #
 # In a previous notebook, we learnt that a linear classifier will define a
 # linear separation to split classes using a linear combination of the input
 # features. In our 2-dimensional space, it means that a linear classifier will
```

Comment:
This is actually more explanation than is in the linear models notebook. I think I already mentioned it in the other PR but it would be nice explain in linear models that the linear separation line formula is a linear combination of the features.

Reply:
Yes indeed, I think is okay to repeats things.
> it would be nice explain in linear models that the linear separation line formula is a linear combination of the features.

+1

Reply:
Sorry i meant that in `linear.py` to include this nice explanation of linear separation.